### DISCUSSION TITLE
Cannot install on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by inthegarage** [apache/couchdb#4278] [Q&A] [answered]
I'm using the instructions mentioned here:

https://docs.couchdb.org/en/3.2.2-docs/install/unix.html

These don't appear to work and at the "sudo apt update" stage, I get the following errors (I've retried this a few times now):
```

Ign:23 https://apache.jfrog.io/artifactory/couchdb-deb una InRelease             
Err:24 https://apache.jfrog.io/artifactory/couchdb-deb una Release
  404  Not Found [IP: 35.82.24.207 443]
Reading package lists... Done
N: Skipping acquisition of configured file 'main/binary-i386/Packages', as repository 'https://download.mono-project.com/repo/ubuntu stable-focal InRelease' doesn't support architecture 'i386'
E: The repository 'https://apache.jfrog.io/artifactory/couchdb-deb una Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
pete@pete-Swift-SF314-42:/etc/apt/sources.list.d$ sudo apt install couchdb
```

The when I try to install:

```
sudo apt install couchdb
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package couchdb is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

```

I read some old posts that "something" wasn't available, but this looks it currently broken. Can anyone advise what I should do instead?

cheers.

**Comment by big-r81** [comment]
Hi,

there is no "Ubuntu 24"... What do you mean?

Can you paste what `source /etc/os-release` outputs?

Can you paste your apt sources list?

**Reply by inthegarage** [reply]
Sorry my bad, typed in a hurry, I've updated it, 20.04.

**Comment by inthegarage** [comment]
I get a new error in apt-update so "something/one" is trying to repair it:

Err:15 https://apache.bintray.com/couchdb-deb focal Release                                                                                                                                                                      
  Could not handshake: The TLS connection was non-properly terminated. [IP: 52.28.39.148 443]

**Comment by inthegarage** [comment]
This is the same issue in a browser so it is a genuine error.

**Comment by big-r81** [comment]
Okay, please do the following:

Run the following commands and paste the output:

1. `cat /etc/os-release`
2. `cat /etc/apt/sources.list.d/couchdb.list`

**Comment by inthegarage** [comment, marked as answer]
No problem, here you go:

os-release:
```
NAME="Linux Mint"
VERSION="20.3 (Una)"
ID=linuxmint
ID_LIKE=ubuntu
PRETTY_NAME="Linux Mint 20.3"
VERSION_ID="20.3"
HOME_URL="https://www.linuxmint.com/"
SUPPORT_URL="https://forums.linuxmint.com/"
BUG_REPORT_URL="http://linuxmint-troubleshooting-guide.readthedocs.io/en/latest/"
PRIVACY_POLICY_URL="https://www.linuxmint.com/"
VERSION_CODENAME=una
UBUNTU_CODENAME=focal
```

couchdb.list:

`deb [signed-by=/usr/share/keyrings/couchdb-archive-keyring.gpg] https://apache.jfrog.io/artifactory/couchdb-deb/ una main`

I see the error has return to 404, so not sure what is happening to the server. However the error is as reported in the 1st message.

**Reply by big-r81** [reply]
Okay, change the word `una` in couchdb.list to `focal` and try again updating `apt`.

**Comment by inthegarage** [comment]
Yes that has worked many thanks @big-r81 

If you wish a bug raised for this then please reply.

**Comment by big-r81** [comment]
@inthegarage Yw!

I'm not sure if it's really a bug, because the documentation stated this steps for Debian/Ubuntu. You are using Linux Mint which is a Ubuntu derivat, but yes please create a bug report and link to this discussion.
